vms/event/tests/test_shiftSignUp.py

```diff
@@ -229,3 +228,142 @@ def test_search_event_end_date_present(self):
         # Verify that the event shows up
         self.assertEqual(sign_up_page.get_event_name(), 'event')
 
+    def test_search_job_name_present(self):
+        """
+        tests for search results on the basis of job name
+        """
+        register_event_utility()
+        register_job_utility()
+        register_shift_utility()
+
+        sign_up_page = self.sign_up_page
+        sign_up_page.live_server_url = self.live_server_url
+        sign_up_page.navigate_to_sign_up()
+        sign_up_page.click_to_view_jobs()
+
+        # Enter name of the job
+        parameters = ['job', '', '', '', '', '']
+        sign_up_page.fill_job_search_form(parameters)
+        # Verify that the job shows up
+        self.assertEqual(sign_up_page.get_job_name(), 'job')
+
+    def test_search_job_both_date_present(self):
+        """
+        tests for search results on the basis of start and end date
+        """
+        register_event_utility()
+        register_job_utility()
+        register_shift_utility()
+
+        sign_up_page = self.sign_up_page
+        sign_up_page.live_server_url = self.live_server_url
+        sign_up_page.navigate_to_sign_up()
+        sign_up_page.click_to_view_jobs()
+
+        # Enter date range in which a job starts
+        parameters = ['', '05/10/2050', '06/15/2050', '', '', '']
+        sign_up_page.fill_job_search_form(parameters)
+        # Verify that the job shows up
+        self.assertEqual(sign_up_page.get_job_name(), 'job')
+
+    def test_search_job_start_date_present(self):
+        """
+        tests for search results on the basis of start date
+        """
+        register_event_utility()
+        register_job_utility()
+        register_shift_utility()
+
+        sign_up_page = self.sign_up_page
+        sign_up_page.live_server_url = self.live_server_url
+        # Enter only correct starting date
+        sign_up_page.navigate_to_sign_up()
+        sign_up_page.click_to_view_jobs()
+
+        parameters = ['', '05/10/2050', '', '', '', '']
+        sign_up_page.fill_job_search_form(parameters)
+        # Verify that the job shows up
+        self.assertEqual(sign_up_page.get_job_name(), 'job')
```

Comment:
Same here?

Reply:
answered above

Reply:
This test is of start date so you should be checking start date right?